iceoryx_posh/include/iceoryx_posh/internal/popo/ports/server_port_user.hpp

```diff
@@ -62,11 +62,18 @@ namespace cxx
 {
 template <>
 constexpr popo::ServerRequestResult
-from<popo::ChunkReceiveResult, popo::ServerRequestResult>(const popo::ChunkReceiveResult result) noexcept;
+from<popo::ChunkReceiveResult, popo::ServerRequestResult>(const popo::ChunkReceiveResult value) noexcept;
 } // namespace cxx
 
 namespace popo
 {
+enum class ServerSendError
+{
+    NOT_OFFERED = 2, // start at 2 to make the C binding match
```

Comment:
Is this somewhere tested? Not that someone is relying on this and we break this by accident.

Reply:
It is not really necessary and a micro optimization. @mossmaurice also asked about this. Maybe it is better to remove it for now and decide later on if we want to go this route for all C binding enums